solidity/contracts/test/TestInterchainGasPaymaster.sol

```diff
@@ -5,15 +5,13 @@ pragma solidity >=0.8.0;
 import {InterchainGasPaymaster} from "../igps/InterchainGasPaymaster.sol";
 
 contract TestInterchainGasPaymaster is InterchainGasPaymaster {
-    uint256 gasPrice = 0;
+    uint256 public constant gasPrice = 10;
```

Comment:
this made testing a lot easier - otherwise in some tests I'd need to connect to the IGP address using `TestInterchainGasPaymaster__factory` in order to set the gas price, which was pretty klunky